02-peer-protocol.md

```diff
@@ -2467,6 +2940,13 @@ fee changes).
     to the ordering of the commitment transaction (see [BOLT #3](03-transactions.md#transaction-input-and-output-ordering)).
   - if it has not recently received a message from the remote node:
       - SHOULD use `ping` and await the reply `pong` before sending `commitment_signed`.
+  - If there are `N` pending splice transactions:
```

Comment:
Could we clarify that `N` MUST be greater than 0 and have the receiver enforce it? Otherwise, a receiver may get `commitment_signed` with a `batch_size` of `1` and use a code path for splicing, which for LDK may have some backwards compatibility concerns.